bitcoin/BRWalletManager.h

```diff
@@ -369,6 +385,30 @@ BRWalletManagerExtractFileServiceTypes (BRFileService fileService,
                                         const char **blocks,
                                         const char **peers);
 
+//
+// Mark: Wallet Sweeper
+//
+
+typedef enum {
+    WALLET_SWEEPER_SUCCESS
+} BRWalletSweeperStatus;
+
+extern BRWalletSweeper // NULL on error
+BRWalletSweeperNew (BRKey *key, uint8_t isSegwit);
```

Comment:
As mentioned previously, maybe the sweeper needs to sweep with both address schemes?

Reply:
The existing iOS functionalty always uses legacy. The isSegwit is used for our received address. We don't store that information anywhere.